### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by maxhutch**
Includes central repo deployment and a random forest convenience wrapper.

### CODE REVIEW

**Comment by maxhutch** [comment]
This deployed to central properly: https://search.maven.org/#search%7Cga%7C1%7Clolo

File: README.md [hunk 1]
```diff
@@ -24,20 +24,22 @@ Lolo supports:
...
+For example, you can use Random Forest with:
+```
+import io.citrine.lolo.learners.RandomForest
+val trainingData: Seq[(Vector[Any], Any)] = features.zip(labels)
+val model = new RandomForest().train(trainingData).getModel()
+val predictions: Seq[Any] = model.transform(testInputs).getExpected()
+```
 
 # Performance
 Lolo prioritizes functionallity over performance, but it is still quite fast.  In its _random forest_ use case, the complexity scales as:
```
Comment (sparadiso) [inline README.md]: functionality (sp)

File: pom.xml [hunk 2]
```diff
@@ -164,44 +241,21 @@
...
-        </site>
-        <repository>
-            <id>s3.release</id>
-            <url>s3://citrine-maven/release</url>
-        </repository>
         <snapshotRepository>
-            <id>s3.snapshot</id>
-            <url>s3://citrine-maven/snapshot</url>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```
Comment (sparadiso) [inline pom.xml]: Will we lose the ability to deploy to our maven repo before it's ready to go to mavn-central? Are we cool with that?

**Comment by maxhutch** [comment]
@sparadiso The pom as-is won't be usable internally, but I can write a little script to swap out the distribution management and repository blocks for internal use.

**Comment by maxhutch** [comment]
Rebuild me!

**Pull request merged**